src/Storage/Storage.Management/StorageAccount/StorageAccountBaseCmdlet.cs

```diff
@@ -225,6 +226,9 @@ public static CloudStorageAccount GetCloudStorageAccount(Track2.StorageAccountRe
             if (storageAccountResource.GetKeys()?.Value?.Keys != null && storageAccountResource.GetKeys().Value.Keys.Count > 0)
             {
                 key = storageAccountResource.GetKeys().Value.Keys[0].Value;
+            } else
+            {
+                throw new InvalidJobStateException("Could not fetch keys.");
```

Comment:
might can change to more detail: Could not fetch storage account keys to build storage account context.